pandas/tests/io/test_packers.py

```diff
@@ -165,12 +165,16 @@ def __init__(self):
                 self.read = 0
 
         msg = "Invalid file path or buffer object type: <class '{}'>"
+        invalid_path = os.path.join("nonexistent_dir", "df.msgpack")
+        path_msg = "File b'{}' does not exist".format(invalid_path)
```

Comment:
just use a regex for 'does not exist' I think this is not matching on windows becase of the escaped pathname.